src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/controllers/advice/AsylumCaseRequestAdapter.java

```diff
@@ -1,38 +0,0 @@
-package uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.controllers.advice;
-
-import java.lang.reflect.Type;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.core.MethodParameter;
-import org.springframework.http.HttpInputMessage;
-import org.springframework.http.converter.HttpMessageConverter;
-import org.springframework.web.bind.annotation.ControllerAdvice;
-import org.springframework.web.context.request.RequestAttributes;
-import org.springframework.web.context.request.RequestContextHolder;
-import org.springframework.web.servlet.mvc.method.annotation.RequestBodyAdviceAdapter;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.CaseDetails;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.ccd.callback.Callback;
-
-@ControllerAdvice
```

Comment:
We need to check this isnt being created as a spring bean and being added to the spring context actually - since it might not be explicitly used but RequestBodyAdviceAdapter is a @Component so will be added to the spring context, and now sure if it will be used by any background spring processes